src/index.js

```diff
@@ -49,7 +50,7 @@ function parseReceiptEvents(abi, address, receipt) {
   Object.keys(receipt.events).forEach(function (n) {
     const event = receipt.events[n]
 
-    if (event.address !== address || event.signature) {
+    if (!sameAddress(event.address, address) || event.signature) {
```

Comment:
Instead of creating a function to compare the addresses, better use `web3Utils.toChecksumAddress()` as follows:

```suggestion
    if (
      web3Utils.toChecksumAddress(event.address) !==
        web3Utils.toChecksumAddress(address) ||
      event.signature
    ) {
```

It is simpler IMO.